technology-considerations/API_management_AppSync_GraphQL_REST.md

```diff
@@ -106,12 +106,12 @@ type Query {
   getFile(id: UUID!): File
 ```
 - API documentation can be generated form the graphql schema [graphdoc](https://github.com/2fd/graphdoc)
-- Intellij can validate types and operations against schema. 
+- Types and operations can be validated against schema with Intellij and during codegen
 
 #####GraphQL enables rapid product development
```

Comment:
Need a space between the hashes and the title as it won't display correctly

Reply:
added space